Docker/Compose/pgsql/MultiTenant/compose-build-dev-multi-tenant.yml

```diff
@@ -30,6 +28,8 @@ services:
       # EdFi.Ods.AdminApi directory, so that the Dockerfile has access to other
       # C# projects.
       context: ../../../
+      additional_contexts:
+        assets: ../../../../
```

Comment:
The MSSQL compose files were already using this `additional_contexts` feature, which is relatively new. And it is a nice feature. It used to be the case that the files we want to copy into the Docker image must be in the same directory (or a child directory) as the Dockerfile. This `additional_contexts` property changes things so that we can now refer directly to the files from elsewhere in the directory structure. 

In this case, it means that we no longer need to copy the `NuGet.config` file and `EdFi.Ods.AdminApi` directory into `Docker`.